Sources/ClientRuntime/Networking/ChecksumAlgorithm.swift

```diff
@@ -40,7 +40,7 @@ public enum ChecksumAlgorithm {
         return hashFunctions
     }
 
-    func toString() -> String {
+    public func toString() -> String {
```

Comment:
Clarification: It looks like the config has ChecksumAlgorithm enum and then the specified value is converted to a string to pass into FlexibleChecksumRequestMiddleware, is that correct?